src/test/resources/2015-1/DTBook.invalid/C00000.invalid.xml

```diff
@@ -44,7 +44,7 @@
     public static void main(String[] args) {
         ...
     }
-</code></p>
+                </code></p>
```

Comment:
same newline issue here, we could maybe try either `xml:space="preserve"` on the `<code>` or `}&#x0A;</code>`

Reply:
newline issue here